apps/nextjs/src/app/[locale]/manage/users/groups/[id]/_reserved-group-alert.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+import { Alert, Anchor } from "@mantine/core";
+import { IconExclamationCircle } from "@tabler/icons-react";
+
+import { getI18n } from "@homarr/translation/server";
+
+export const ReservedGroupAlert = async () => {
+  const t = await getI18n();
+
+  return (
+    <Alert variant="light" color="yellow" icon={<IconExclamationCircle size="1rem" stroke={1.5} />}>
+      {t("group.reservedNotice.message", {
+        checkoutDocs: (
+          <Anchor
+            size="sm"
+            component={Link}
+            href="https://deploy-preview-113--homarr-docs.netlify.app/docs/management/users/#special-groups"
```

Comment:
**NEVER** do this. It will **always** be forgotten.
Introduce a constant and use it as a base for all links. Additionally, I'm unsure whether we should use the correct URL here or not. We'll probably forget to set this to ``https://homarr.dev`` when we publish 1.0